docs/dev-cloudtrail-initial-setup.rst

```diff
@@ -0,0 +1,112 @@
+***********************************
+AWS S3 and SQS setup for CloudTrail
+***********************************
+
+S3 setup
+========
+#. log in to the AWS web console for your cluster account
+#. go to the s3 (Simple Storage Service) page
+#. click "Create bucket"
+#. enter a unique name for your bucket ("yourname-cloudigrade-s3")
+#. select a region (us-east-1) and click "Next"
+#. the default properties and permissions are OK, click "Next" until you reach the "Review" step
+#. click "Create bucket"
+
+SQS setup
+=========
+#. navigate to the SQS (Simple Queue Service) page
+#. click "Create New Queue"
+#. give the queue a reasonably unique name ("yourname-cloudigrade-sqs")
+#. select Standard Queue and click "Quick-Create Queue"
+#. select your queue and click on the "Permissions" tab
+#. select "Edit Policy Document (Advanced)"
+#. click JSON to switch to the text editor.
+#. replace the contents of the text editor with the following:
+
+   .. code-block:: json
+
+       {
+          "Version": "2012-10-17",
+          "Id": "unique-identifier",
+          "Statement": [
+            {
+              "Sid": "unique-identifier",
+              "Effect": "Allow",
+              "Principal": {
+                "AWS": "*"
+              },
+              "Action": "SQS:SendMessage",
+              "Resource": "arn-for-your-queue",
```

Comment:
Maybe include a step above to describe where to get the ARN for your queue. Add a step that says to copy and keep it for later.